components/ingest-service/migration/actions_migration.go

```diff
@@ -0,0 +1,143 @@
+package migration
```

Comment:
This code walks through each of the chef actions in the "actions-*" indexes and re-ingest them through the pipeline to send them to the event-feed-service and the policy information to the config-mgmt-service.